docs/data/joy/customization/theme-tokens/theme-tokens.md

```diff
@@ -0,0 +1,351 @@
+# Theme tokens
+
+<p class="description">Learn how to customize the theme tokens.</p>
+
+<!-- This page should answer: -->
+<!-- What's the structure -->
+<!-- How to override, and how to augment the type -->
+<!-- (optional) live playground -->
+
+## Override default tokens
```

Comment:
I was expecting this page to also cover in more details the `theme.vars` but it is not described even here.